docs/kubernetes/ingress-usage.md

```diff
@@ -4,21 +4,21 @@ This documentation is meant for people deploying to Kubernetes
 Clusters and describes to use Ingress and low level and high level
 features Skipper provides.
 
-[RouteGroups](routegroups.md), a relatively new feature, also
-support each of these features, with an alternative format that
-supports them in a more native way. The documentation contains a
-section with
-[mapping](routegroups.md#mapping-from-ingress-to-routegroups)
-Ingress to RouteGroups.
+[RouteGroups](routegroups.md), skipper Kubernetes native routing
```

Comment:
Maybe spell Skipper and Ingress from capital letter here and elsewhere

Reply:
And Kubernetes as well.

Reply:
done